etna/transforms/math/limit.py

```diff
@@ -0,0 +1,122 @@
+from typing import List
+import numpy as np
+import pandas as pd
+
+from etna.datasets import TSDataset
+from etna.datasets import set_columns_wide
+from etna.transforms.base import ReversibleTransform
+from etna.transforms.utils import match_target_quantiles
+
+
+class LimitTransform(ReversibleTransform):
+    """LimitTransform limits values of some feature between the borders"""
+
+    def __init__(self, in_column: str, lower_bound: float = -1e10, upper_bound: float = 1e10):
+        """
+        Init LimitTransform.
+
+        Parameters
+        ----------
+        in_column:
+            column to apply transform
+        lower_bound:
+            lower bound for the value of the column
+        upper_bound:
+            upper bound for the value of the column
+        """
+        super().__init__(required_features=[in_column])
+        self.in_column = in_column
+        self.lower_bound = lower_bound
+        self.upper_bound = upper_bound
+        self.tol = 1e-10
+
+    def _get_column_name(self) -> str:
```

Comment:
You don't actually need this method, this method is implemented in `AddConstTransform` to handle output column name generation logic. Basically you need to implement only te abstract methods from the base class